deploy/sechack/.env.local

```
@@ -0,0 +1,24 @@
+# D-TPRES Local AO Environment - AUTO GENERATED
+NETWORK=local
+PROTOCOL=ao
+VARIANT=ao.TN.1
+
+# AO Network URLs
+ARWEAVE_URL=http://localhost:1984
+GRAPHQL_URL=http://localhost:1984/graphql
+MU_URL=http://localhost:1985
+SU_URL=http://localhost:1986
+CU_URL=http://localhost:1987
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CU_URL key should go before the GRAPHQL_URL key

(UnorderedKey)